examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -786,16 +788,17 @@ def periodic_spline_transformer(period, n_splines=None, degree=3):
     "Splines + polynomial kernel",
     "Gradient Boosted Trees",
 ]
-for ax, pred, label in zip(axes, predictions, labels):
-    ax.scatter(y.iloc[test_0].values, pred, alpha=0.3, label=label)
-    ax.plot([0, 1], [0, 1], "--", label="Perfect model")
-    ax.set(
-        xlim=(0, 1),
-        ylim=(0, 1),
-        xlabel="True demand",
-        ylabel="Predicted demand",
-    )
-    ax.legend()
+kind = ["actual_vs_predicted", "residual_vs_predicted"]
+for i in range(2):
```

Comment:
It would be more explicit to have:

```python
for axis_idx, kind in enumerate(plot_kinds):
    ... # passing `kind` directly
```

Here, I would rename the current `kind` to `plot_kinds` to iterate on it.